src/mainboard/clevo/tgl-u/ramstage.c

```diff
@@ -112,11 +113,42 @@ static void set_fan_curve(void)
 	}
 }
 
+static void set_wifi_bt_radios(void)
+{
+    bool enable = true;
+
+#if CONFIG(DRIVERS_EFI_VARIABLE_STORE)
+    struct region_device rdev;
+    enum cb_err ret;
+
+    uint32_t size = sizeof(enable);
+
+    const EFI_GUID dasharo_system_features_guid = {
+		0xd15b327e, 0xff2d, 0x4fc1, { 0xab, 0xf6, 0xc1, 0x2b, 0xd0, 0x8c, 0x13, 0x59 }
+	};
+
+    if (smmstore_lookup_region(&rdev))
+		goto efi_err;
+
+    ret = efi_fv_get_option(&rdev, &dasharo_system_features_guid, "EnableWifiBt",
+                            &enable, &size);
+
+    if (ret != CB_SUCCESS)
+            printk(BIOS_DEBUG, "Wifi + BT radios: failed to read curve selection from EFI vars, using board default\n");
+efi_err: ;
+#endif
+	struct device *wifi_card = pcidev_on_root(0x14, 3);
+	wifi_card->enabled = enable;
+	printk(BIOS_DEBUG, enable ? "raa Enabled\n" : "raa disabled\n");
```

Comment:
```suggestion
	printk(BIOS_DEBUG, enable ? "raa Enabled\n" : "raa disabled\n");
```

what is `raa`?

Reply:
just debug print, removed in 31f28991bb1c1c1425e8a81e98819326d217a534